### PR TITLE
chore: add analytics for dapp shortcuts

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

As the title

### Test plan

n/a

### Related issues

- Fixes RET-733

### Backwards compatibility

Y

### CODE REVIEW

File: src/analytics/Properties.tsx [hunk 1]
```diff
@@ -1311,6 +1312,27 @@ interface BuilderHooksProperties {
...
+  rewardId: string
+  appName: string
+}
+interface DappShortcutsProperties {
+  [DappShortcutsEvents.dapp_shortcuts_rewards_screen_open]: {
+    numRewards: number
+  }
+  [DappShortcutsEvents.dapp_shortcuts_reward_claim_start]: DappShortcutClaimRewardEvent & {
+    rewardTokens: string[]
+    rewardAmounts: string[]
```
Comment (jeanregisser) [inline src/analytics/Properties.tsx:1325]: I forgot if arrays are easy to analyze in our analytics pipeline.

File: src/analytics/Properties.tsx [hunk 2]
```diff
@@ -1311,6 +1312,27 @@ interface BuilderHooksProperties {
...
+    shortcutId: string
```
Comment (jeanregisser) [inline src/analytics/Properties.tsx:1326]: Can we also add `claimableValueUsd`?

File: src/analytics/Properties.tsx [hunk 3]
```diff
@@ -1311,6 +1312,27 @@ interface BuilderHooksProperties {
   [BuilderHooksEvents.hooks_disable_preview]: undefined
 }
 
+interface DappShortcutClaimRewardEvent {
+  rewardId: string
+  appName: string
+}
```
Comment (jeanregisser) [inline src/analytics/Properties.tsx:1318]: Can we add the following properties?
- `appId`
- `network`
- `shortcutId`

**Review by jeanregisser — APPROVED** [review]
Awesome! 🚀 

Already approve with some small suggestions.

File: src/analytics/Properties.tsx (hunk 1 above)
Comment (kathaypacific) [reply src/analytics/Properties.tsx]: i'm also not sure, i asked the question in the #data channel. i changed these to csv's to be safe, although i do see us using string[] elsewhere

File: src/analytics/Properties.tsx (hunk 2 above)
Comment (kathaypacific) [reply src/analytics/Properties.tsx]: yes, done!

**Pull request merged**